### PR TITLE
Add tech demo presentation

### PR DESCRIPTION
**Pull request by Felixmil**
rendered version: [technical-demo-presentation.zip](https://github.com/user-attachments/files/16691772/technical-demo.zip)

### CODE REVIEW

File: vignettes/articles/presentations/technical-demo/technical-demo.qmd [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+Develop a clinical trial simulator (**CTS**) for drug-drug interactions (DDI) for contraceptives using physiologically based pharmacokinetic (PBPK) models snapshots.
+
+### 📍 Scope
+
+-   R package
+-   Programming Interface
+
+### 📅 Deadline
+
+31th October 2024
```
Comment (PavelBal) [inline vignettes/articles/presentations/technical-demo/technical-demo.qmd:47]: Mid october

File: vignettes/articles/presentations/technical-demo/technical-demo.qmd [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+# ✅Requirements {.smaller}
+
+## 👥 User Requirements
+
+-   Easily install the package,
+-   Browse and select compounds from the OSP model library,
+-   Import compounds snapshots from a json file path or a url,
+-   Review the compound model parameters,
+-   Review the DDI model parameters,
+-   Parameterize some aspects of the DDI simulation
```
Comment (PavelBal) [inline vignettes/articles/presentations/technical-demo/technical-demo.qmd:93]: "Define study design" - i.e., define application protocols and apply them

File: vignettes/articles/presentations/technical-demo/technical-demo.qmd [hunk 3]
```diff
@@ -0,0 +1,202 @@
...
+-   Export the DDI model as a Project Snapshot,
+-   Import a DDI model snapshot,
```
Comment (PavelBal) [inline vignettes/articles/presentations/technical-demo/technical-demo.qmd:95]: You mean from a DDI repository on GitHub, like this? https://github.com/Open-Systems-Pharmacology/Fluvoxamine-Quinidine-DDI

No this is not required.

**Review by PavelBal — CHANGES_REQUESTED** [review]
Very nice.

Small minor comments.

File: vignettes/articles/presentations/technical-demo/technical-demo.qmd (hunk 3 above)
Comment (Felixmil) [reply vignettes/articles/presentations/technical-demo/technical-demo.qmd:95]: No, I mean reimporting a DDI model snapshot that was created with this tool before. For example, to change study design or other parameter before re-exporting it.

File: vignettes/articles/presentations/technical-demo/technical-demo.qmd (hunk 1 above)
Comment (Felixmil) [reply vignettes/articles/presentations/technical-demo/technical-demo.qmd:47]: Aligned with Laura Kata, it's actually end of October (We shifted this few weeks ago to take into account the already planned vacations). It is updated on Monday too.

**Comment by Felixmil** [comment]
New rendered version: 
[technical-demo.zip](https://github.com/user-attachments/files/16759584/technical-demo.zip)

**Review by PavelBal — APPROVED** [review]

**Pull request merged**